src/main/io/serial_cli.c

```diff
@@ -499,7 +508,11 @@ const clivalue_t valueTable[] = {
     { "enable_buzzer_p6",           VAR_UINT8  | MASTER_VALUE | MODE_LOOKUP,  &masterConfig.use_buzzer_p6, .config.lookup = { TABLE_OFF_ON } },
 #endif
     { "motor_pwm_rate",             VAR_UINT16 | MASTER_VALUE,  &masterConfig.motor_pwm_rate, .config.minmax = { 50,  32000 } },
+    { "motor_pwm_protocol",         VAR_UINT8  | MASTER_VALUE | MODE_LOOKUP,  &masterConfig.motor_pwm_protocol, .config.lookup = { TABLE_PWM_PROTOCOL } },
     { "servo_pwm_rate",             VAR_UINT16 | MASTER_VALUE,  &masterConfig.servo_pwm_rate, .config.minmax = { 50,  498 } },
+#ifdef BRUSHED_MOTORS 
```

Comment:
Why this is conditional compiled? Should be possible to enable brushed motors for any target?